docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+## Pull requests & issues
+
+We welcome any contributions to this project.
+
+If you have a small bugfix and already have the code available, feel free to [create a pull request](https://github.com/Automattic/vip-design-system/compare). If you have a feature suggestion, please [create an issue](https://github.com/Automattic/vip-design-system/issues/new). Please follow the instructions provided. Please assign a priority to the issue/pull request according to the definitions found below.
```

Comment:
The priority labels are missing. It is up to you, but I feel that we should have them listed everywhere so to have consistency across projects in how they are perceived and applied.

Reply:
We don't have this label-system organized in the vip-design-system repository, is there a P2 explaining how they should work? I excluded because I didn't know if this was a thing related to the vip-go-crontab